lib/ansible/modules/cloud/azure/azure_rm_postgresqlconfiguration_facts.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, (@zikalino)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_postgresqlconfiguration_facts
+version_added: "2.8"
+short_description: Get Azure PostgreSQL Configuration facts.
+description:
+    - Get facts of Azure PostgreSQL Configuration.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    server_name:
+        description:
+            - The name of the server.
+        required: True
+    setting_name:
+        description:
+            - Setting name.
+
+extends_documentation_fragment:
+    - azure
+
+author:
+    - "Zim Kalinowski (@zikalino)"
+
+'''
+
+EXAMPLES = '''
+  - name: Get specific setting of PostgreSQL configuration
+    azure_rm_postgresqlconfiguration_facts:
+      resource_group: testrg
+      server_name: testpostgresqlserver
+      setting_name: deadlock_timeout
+
+  - name: Get all settings of PostgreSQL Configuration
+    azure_rm_postgresqlconfiguration_facts:
+      resource_group: resource_group_name
+      server_name: server_name
+'''
+
+RETURN = '''
+settings:
+    description: A list of dictionaries containing MySQL Server settings.
+    returned: always
+    type: complex
+    contains:
+        id:
+            description:
+                - Setting resource ID
+            returned: always
+            type: str
+            sample: "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testrg/providers/Microsoft.DBforPostgreSQL/servers/testpostgresqlser
```

Comment:
xxxx-xxxx ... for consistent?

Reply:
yes, i think we should agree on common "placeholder" value, now we have a few different....